src/iomei.cpp

```diff
@@ -5031,6 +5031,20 @@ bool MEIInput::ReadCustos(Object *parent, pugi::xml_node custos)
     ReadPositionInterface(custos, vrvCustos);
     vrvCustos->ReadColor(custos);
 
+    // copied from ReadNote -- this should be moved to a common method
+    AttAccidental accidental;
+    accidental.ReadAccidental(custos); 
+    AttAccidentalGestural accidentalGestural;
+    accidentalGestural.ReadAccidentalGestural(custos);
+    if (accidental.HasAccid() || accidentalGestural.HasAccidGes()) {
+        Accid *vrvAccid = new Accid();
+        vrvAccid->IsAttribute(true);
+        vrvAccid->SetAccid(accidental.GetAccid());
+        vrvAccid->SetAccidGes(accidentalGestural.GetAccidGes());
+        vrvCustos->AddChild(vrvAccid);
+    }
+    // -- end copy
```

Comment:
Could you make a common `MEIInput::ReadAccidAttr(pugi::xml_node node, Object *object)` to be place there https://github.com/rism-digital/verovio/blob/7a392176fc9d49b97c0c6fac555113e7ffc0311e/include/vrv/iomei.h#L692-L701 ? Thanks